lib/ptor.ts

```diff
@@ -23,7 +23,7 @@ export class Ptor {
   By: ProtractorBy;
   by: ProtractorBy;
   wrapDriver:
-      (webdriver: webdriver.WebDriver, baseUrl?: string, rootElement?: string,
+      (webdriver: WebDriver, baseUrl?: string, rootElement?: string,
        untrackOutstandingTimeouts?: boolean) => ProtractorBrowser;
   ExpectedConditions: ProtractorExpectedConditions;
 
```

Comment:
nit: below, for everything under `// Export selenium webdriver.`, prefer the imported version vs the `webdriver.` version